### DISCUSSION TITLE
zh/faq/howto

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#54] [Announcements]
# zh/faq/howto

AList v3的文档

https://alist.nn.ci/zh/faq/howto.html

**Comment by zhuangyin8** [comment]
离线下载 添加磁力链接提示 Aria2 not ready

**Comment by zero617** [comment]
登陆太多次失败怎么重置？

**Reply by wyf9** [reply]
上面 `为什么` 里有标题: `Too many unsuccessful sign-in attempts have been made using an incorrect username or password, Try again later.`

解决方法：重启Alist服务

https://alist.nn.ci/zh/faq/why.html#too-many-unsuccessful-sign-in-attempts-have-been-made-using-an-incorrect-username-or-password-try-again-later

**Comment by zhlx2835** [comment]
如何配置IPA在线安装

**Comment by lightsky9627** [comment]
挂载中国移动云盘，上传格式为pdf的文件。显示签名校验失败。
[31mERRO[0m[2022-11-22 21:54:15] failed put /移动云盘/books/: 签名校验失败

**Comment by xztsummer** [comment]
请问如何关闭“双重身份验证”？？？开了之后嫌登录麻烦，但是没有找到入口可以关闭

**Reply by xztsummer** [reply]
已解决。
windos：alist cancel2fa
or
Linux：./alist cancel2fa

**Comment by 1264967889** [comment]
电影播放失败，但是其他的播放器能正常播放

**Comment by zhaoxp1js** [comment]
谁可以帮我重置一下Alist的密码

**Reply by wyf9** [reply]
```
root@onecloud:/opt/alist# ./alist admin
INFO[2024-08-11 14:18:45] reading config file: data/config.json        
INFO[2024-08-11 14:18:45] load config from env with prefix: ALIST_     
INFO[2024-08-11 14:18:45] init logrus...                               
INFO[2024-08-11 14:18:45] Admin user's username: admin                 
INFO[2024-08-11 14:18:45] The password can only be output at the first startup, and then stored as a hash value, which cannot be reversed 
INFO[2024-08-11 14:18:45] You can reset the password with a random string by running [alist admin random] 
INFO[2024-08-11 14:18:45] You can also set a new password by running [alist admin set NEW_PASSWORD]
```

如上，可以用 `alist admin random` 设置随机密码，或者用 `alist admin set 新密码` 设置密码

**Reply by wyf9** [reply]
还有这页就有: `忘记密码怎么办？`

https://alist.nn.ci/zh/faq/howto.html#%E5%BF%98%E8%AE%B0%E5%AF%86%E7%A0%81%E6%80%8E%E4%B9%88%E5%8A%9E

**Comment by Mubai1111** [comment]
invalid X-Device-Id 如何解决

**Comment by crazyboy945** [comment]
用windows “计划任务程序”设置自动启动后，无法登录，提示用户密码错误！

**Reply by wyf9** [reply]
可能是配置文件的路径不对？

试试把 任务计划-操作-启动程序 里的 `起始于` 改成你alist程序所在的目录

**Reply by wyf9** [reply]
官方回答：
侧边栏这页上方 `为什么` 里有: `为什么我通过在终端输入./alist admin获取的密码还是不正确？`

https://alist.nn.ci/zh/faq/why.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E6%88%91%E9%80%9A%E8%BF%87%E5%9C%A8%E7%BB%88%E7%AB%AF%E8%BE%93%E5%85%A5-alist-admin%E8%8E%B7%E5%8F%96%E7%9A%84%E5%AF%86%E7%A0%81%E8%BF%98%E6%98%AF%E4%B8%8D%E6%AD%A3%E7%A1%AE

**Comment by chinattw** [comment]
请问：alist怎么配置自签名的https或wss的aria2地址,直接输入https://本机aria2 或 wss://本机aria2地址 时，报证书错误。

**Comment by wangyan0001** [comment]
输入密码老是提示错误。   请给与重置密码。谢谢！hao37006388@163.com

**Comment by Hooog0207** [comment]
群晖套件中心安装的 命令提示找不到目录压 得加啥  忘记密码了

**Comment by youtehub** [comment]
ocr接口试试这个，刚部署的还没测试。
https://ocr-dwxg.onrender.com/ocr/file/json
填写到后台--设置--全局--OCR接口，添加后可以直接使用

**Comment by hqyun4020** [comment]
用脚本安装的AList,IPv6访问不了了

**Comment by JustinBridge** [comment]
需要重置密码

**Comment by Aggnem** [comment]
需要重置密码

**Reply by cxapk** [reply]
重置alist试一试

**Comment by weiniduai** [comment]
密码是多少啊。。

**Reply by cxapk** [reply]
啥密码?

**Comment by a841603239** [comment]
顶

**Comment by a841603239** [comment]
支持

**Comment by cq-cppc** [comment]
密码是什么

**Comment by q361682172** [comment]
极空间docker怎么找回alist的账号密码啊谢谢！！361682172@qq.com

**Comment by nlfnb** [comment]
密码是什么？

**Comment by mkelaolang** [comment]
./alist admin      忘记密码了怎么重置啊

**Reply by wyf9** [reply]
```
./alist admin set 新密码
```

**Comment by xiaoyan94** [comment]
## 😄Docker或者 Docker-Compose 部署的用户，重置密码的方法：

1. Docker-compose部署，假设服务容器名是 alist，`docker-compose exec alist` 后跟执行重置密码的命令`./alist admin set 1234567890`

```bash
docker-compose exec alist ./alist admin set 1234567890
```

2. Docker 部署，同样地

```bash
docker exec alist ./alist admin set 1234567890
```

**Comment by 2977505400** [comment]
mkv格式视频预览能否加载内封字幕

**Comment by yaozhy7** [comment]
输入密码显示错误，请求重置密码 
 1607514733@qq.com

**Comment by 15079505721** [comment]
怎么注册账号啊
